### PR TITLE
Beta/integrate vault

### PR DESCRIPTION
**Pull request by mcjcloud**
Integrates vault into the Account creation process. Also includes fixes for UpdateWhoIs. Dev was merged in.

What I changed in this PR:
* internal/motor
* pkg/did
* pkg/vault
* x/registry

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: cmd/highway-cli/main.go [hunk 1]
```diff
@@ -28,7 +28,6 @@ var (
 
 // load environment variables
 func loadEnv() error {
-	viper.AddConfigPath(CONFIG_PATH_LOCAL)
```
Comment (ntindle) [inline cmd/highway-cli/main.go:31]: ?

File: internal/motor/config.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 
 func setupDefault() (*MotorNode, error) {
 	// Create Client instance
-	c := client.NewClient(client.ConnEndpointType_BETA)
+	c := client.NewClient(client.ConnEndpointType_LOCAL)
```
Comment (ntindle) [inline internal/motor/config.go]: Can we read this from env?

Comment (ntindle) [reply internal/motor/config.go]: or config or something

File: pkg/jwt/config.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+var (
+	options JWTOptions = JWTOptions{}
+)
+
+func (opts *JWTOptions) DefaultTestConfig() JWTOptions {
+
+	return JWTOptions{
+		// using rsa pub key as secret
+		secret:        []byte("MIICXAIBAAKBgQCGcXG10nJ5I7OCma6hoH3VMv9OmVV4f8DpSd3Z1I2ud4GpBKFRgCs3AeGLOF87qpGdPh6lCugH3IihKy+vuhISpiploBNyGXEAdHbKyNOCHKvEuOxe9MUEE5oLYvE8qv5LPq5G971PC1HGCwPCglxLXHxzAUnOYsJkFekoWeeBIQIDAQABAoGAMVUPVJiUSL9A73tVCRnLEqBT7pN1OXInZ3MjZPsJwis3+L0qNK3DLbwS9vMIfuxn4jsZI5aM7dWOjRU7uk+csZKPqlvvPyToq8eEBcXWz5R2XpIvzrvucVI4vZMKGgZi8YlzvLnTqdfaGy/AqFDqvmU7fX/aQIu+8bC+DZ1XP8ECQQDJG07O+7hPNtsAAC4lATuIx75SCn+9tHj11I3X7pSV3em+co3XmTYIJsX/G2ehArbUyg5ED8OvjbfxD5qCQT03AkEAqyPm1OhtcI1tq9JGsM1y6av0mM4atEjHvsrxaN0Lt9HyUP8xD52ARxYEilVytYsnArzBGv5sZz7rVpZ2jocgZwJAPjD2tyW7Aqw5H4/utTzjV1JF9gMPK/Bis8surkc2pf4Bagbs/G6B+hVbh5/G9VDsj3OI491oK6MM7jxgEMXyEwJACwZvEnw+wKd7zzvmrfEuW/tl8IomkkK2C4aLctP6s0blM26dPIJLB0lV1YuXrjZetwBt+E03spcNFjDvRlNSNQJBAKd1d0+nm2ruD6vmw7tmJQ/iFIuKRMtcTKd2nVhO7TCw7ORM4tH56NWj74uioXQ1YmfFHvwZY3O0woLTa91scYk="),
```
Comment (ntindle) [inline pkg/jwt/config.go:21]: ?????????????????????????????????????????

File: cmd/highway-cli/main.go (hunk 1 above)
Comment (mcjcloud) [reply cmd/highway-cli/main.go:31]: From Josh's changes in dev

File: internal/motor/config.go (hunk 2 above)
Comment (mcjcloud) [reply internal/motor/config.go]: actually did not mean to make this change, but we should have this be configurable before `dev`.

File: pkg/jwt/config.go (hunk 3 above)
Comment (mcjcloud) [reply pkg/jwt/config.go:21]: Also not mine.. I think this is just for testing in jwx stuff.

**Pull request merged**